DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Model/PerturbSupportedVR .cs

```diff
@@ -0,0 +1,12 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors.Model
+{
+    public enum RefreshUIDSupportedVR
```

Comment:
There are mistakes between class names and file names... See perturbSupporterdVR and RefershUidSupportedVR

Reply:
Fixed it.